src/arcaflow_plugin_sdk/test_plugin.py

```diff
@@ -4,30 +4,30 @@
 import typing
 import unittest
 
-from arcaflow_plugin_sdk import plugin
+from arcaflow_plugin_sdk import plugin, schema
 
 
 @dataclasses.dataclass
-class StdoutTestInput:
+class EmptyTestInput:
     pass
 
 
 @dataclasses.dataclass
-class StdoutTestOutput:
+class EmptyTestOutput:
     pass
 
 
 @plugin.step(
     "stdout-test",
     "Stdout test",
     "A test for writing to stdout.",
-    {"success": StdoutTestOutput},
+    {"success": EmptyTestOutput},
 )
 def stdout_test_step(
-    input: StdoutTestInput,
-) -> typing.Tuple[str, StdoutTestOutput]:
+    _: EmptyTestInput,
+) -> typing.Tuple[str, EmptyTestOutput]:
```

Comment:
It seems weird (and overdone) to create a specific type for an input that we're going to ignore.  I would have thought that `Any` would suffice.

Ditto line 62.